src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Server/Properties/launchSettings.json

```diff
@@ -15,7 +15,6 @@
       "launchBrowser": true,
       //#if (useWasm)
       "launchUrl": "",
-      "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
```

Comment:
Let's keep this, wasm debugging will be able to use this through the server app as well (it's not working yet but will soon).